core/src/main/resources/error/error-classes.json

```diff
@@ -1473,6 +1493,11 @@
       "The url is invalid: <url>. If necessary set <ansiConfig> to \"false\" to bypass this error."
     ]
   },
+  "INVALID_VIEW_TEXT" : {
+    "message" : [
+      "The view <tableName> cannot be displayed due to invalid view text: <viewText>. This may be caused by an unauthorized modification of the view or an incorrect query syntax. Please check your query syntax and verify that the view has not been tampered with."
```

Comment:
```suggestion
      "The view <viewName> cannot be displayed due to invalid view text: <viewText>. This may be caused by an unauthorized modification of the view or an incorrect query syntax. Please check your query syntax and verify that the view has not been tampered with."
```